diff/apply/apply_linux.go

```diff
@@ -26,11 +26,16 @@ import (
 	"github.com/containerd/containerd/archive"
 	"github.com/containerd/containerd/errdefs"
 	"github.com/containerd/containerd/mount"
+	"github.com/opencontainers/runc/libcontainer/system"
 	"github.com/pkg/errors"
 )
 
 func apply(ctx context.Context, mounts []mount.Mount, r io.Reader) error {
 	switch {
+	// OverlayConvertWhiteout (mknod c 0 0) doesn't work in userns.
+	// https://github.com/containerd/containerd/issues/3762
+	case system.RunningInUserNS():
+		break
```

Comment:
Should we just do the break right at the beginning of the overlay case? AUFS should still work fine right?

Reply:
I don't think AUFS work in userns

Reply:
it seems AUFS can support userns with modprobe option, though I feel AUFS driver is substantially being deprecated 

http://aufs.sourceforge.net/aufs4/man.html

Reply:
updated